libs/cli/src/generators/base/versions.ts

```diff
@@ -3,27 +3,27 @@ export const FALLBACK_ANGULAR_VERSION = '^18.0.0';
 // ng-icon dependency
 export const NG_ICONS_VERSION = '^29.10.0';
 // spartan dependencies
-export const SPARTAN_ACCORDION_BRAIN_VERSION = '0.0.1-alpha.357';
-export const SPARTAN_ALERT_DIALOG_BRAIN_VERSION = '0.0.1-alpha.357';
-export const SPARTAN_AVATAR_BRAIN_VERSION = '0.0.1-alpha.357';
-export const SPARTAN_COMMAND_BRAIN_VERSION = '0.0.1-alpha.357';
-export const SPARTAN_COLLAPSIBLE_BRAIN_VERSION = '0.0.1-alpha.357';
-export const SPARTAN_CORE_VERSION = '0.0.1-alpha.357';
-export const SPARTAN_DIALOG_BRAIN_VERSION = '0.0.1-alpha.357';
-export const SPARTAN_FORMFIELD_VERSION = '0.0.1-alpha.357';
-export const SPARTAN_HOVERCARD_BRAIN_VERSION = '0.0.1-alpha.357';
-export const SPARTAN_LABEL_BRAIN_VERSION = '0.0.1-alpha.357';
-export const SPARTAN_MENU_BRAIN_VERSION = '0.0.1-alpha.357';
-export const SPARTAN_POPOVER_BRAIN_VERSION = '0.0.1-alpha.357';
-export const SPARTAN_PROGRESS_BRAIN_VERSION = '0.0.1-alpha.357';
-export const SPARTAN_RADIO_GROUP_BRAIN_VERSION = '0.0.1-alpha.357';
-export const SPARTAN_SELECT_BRAIN_VERSION = '0.0.1-alpha.357';
-export const SPARTAN_SEPARATOR_BRAIN_VERSION = '0.0.1-alpha.357';
-export const SPARTAN_SHEET_BRAIN_VERSION = '0.0.1-alpha.357';
-export const SPARTAN_SWITCH_VERSION = '0.0.1-alpha.357';
-export const SPARTAN_TABS_VERSION = '0.0.1-alpha.357';
-export const SPARTAN_TABLE_VERSION = '0.0.1-alpha.357';
-export const SPARTAN_TOGGLE_VERSION = '0.0.1-alpha.357';
+export const SPARTAN_ACCORDION_BRAIN_VERSION = '0.0.1-alpha.381';
```

Comment:
sorry, did not think of this before, but is this still useful?
don't we just have only one brain version?

Reply:
good point! I think we can remove a bunch of complexity with versions as we just depend on the single brain package!